sdk/core/core-rest-pipeline/src/httpHeaders.ts

```diff
@@ -84,6 +84,6 @@ class HttpHeadersImpl implements HttpHeaders {
  * Creates an object that satisfies the `HttpHeaders` interface.
  * @param rawHeaders - A simple object representing initial headers
  */
-export function createHttpHeaders(rawHeaders?: RawHttpHeaders): HttpHeaders {
+export function createHttpHeaders(rawHeaders?: RawHttpHeaders | RawHttpHeadersInput): HttpHeaders {
```

Comment:
Since `RawHttpHeadersInput` is a superset of `RawHttpHeaders` I don't think we need both

```suggestion
export function createHttpHeaders(rawHeaders?: RawHttpHeadersInput): HttpHeaders {
```